ddtrace/appsec/_iast/_taint_tracking/__init__.py

```diff
@@ -94,7 +94,7 @@ def iast_taint_log_error(msg):
 
         stack = inspect.stack()
         frame_info = "\n".join("%s %s" % (frame_info.filename, frame_info.lineno) for frame_info in stack[:7])
-        log_message = "%s:\n%s", msg, frame_info
+        log_message = "%s:\n%s" % (msg, frame_info)
```

Comment:
I think this disables the lazy evaluation in logging no?
https://docs.python.org/3/howto/logging.html#logging-variable-data